src/powershell-native/nativemsh/pwrshcommon/NativeMsh.h

```diff
@@ -1,7 +1,7 @@
 // ----------------------------------------------------------------------
 //
 //  Microsoft Windows NT
-//  Copyright (C) Microsoft Corporation, 2005.
+//  Copyright (c) Microsoft Corporation, 2005.
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."